<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hysettingsBase</class>
 <widget class="QWidget" name="hysettingsBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>10</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="hyphenationTab">
      <attribute name="title">
       <string>Hyphenation</string>
      </attribute>
      <layout class="QGridLayout" name="hyphGridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>General Options</string>
         </property>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="margin">
           <number>10</number>
          </property>
          <item>
           <widget class="QCheckBox" name="verbose">
            <property name="toolTip">
             <string>A dialog box showing all possible hyphens for each word will show up when you use the Extras, Hyphenate Text option.</string>
            </property>
            <property name="text">
             <string>&amp;Hyphenation Suggestions</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="input">
            <property name="toolTip">
             <string>Enables automatic hyphenation of your text while typing.</string>
            </property>
            <property name="text">
             <string>Hyphenate Text Automatically &amp;During Typing</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Behaviour</string>
         </property>
         <layout class="QGridLayout">
          <property name="margin">
           <number>10</number>
          </property>
          <property name="spacing">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&amp;Language:</string>
            </property>
            <property name="buddy">
             <cstring>language</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="QComboBox" name="language"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&amp;Smallest Word:</string>
            </property>
            <property name="buddy">
             <cstring>wordLen</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="wordLen">
            <property name="toolTip">
             <string>Length of the smallest word to be hyphenated.</string>
            </property>
            <property name="minimum">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2" colspan="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Chars</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Consecutive Hyphenations &amp;Allowed:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>maxCount</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QSpinBox" name="maxCount">
            <property name="toolTip">
             <string>Maximum number of Hyphenations following each other.
A value of 0 means unlimited hyphenations.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Exceptions</string>
         </property>
         <layout class="QGridLayout">
          <property name="margin">
           <number>10</number>
          </property>
          <property name="spacing">
           <number>5</number>
          </property>
          <item row="0" column="0" colspan="4">
           <widget class="QListWidget" name="exceptList"/>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="buttonExceptAdd">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="buttonExceptEdit">
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="buttonExceptRemove">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Ignore List</string>
         </property>
         <layout class="QGridLayout">
          <property name="margin">
           <number>10</number>
          </property>
          <property name="spacing">
           <number>5</number>
          </property>
          <item row="0" column="0" colspan="4">
           <widget class="QListWidget" name="ignoreList">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="buttonIgnoreAdd">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="buttonIgnoreEdit">
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="buttonIgnoreRemove">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>21</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="spellingTab">
      <attribute name="title">
       <string>Spelling</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Installed Spelling Dictionaries</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="dictTableWidget">
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Downloadable Spelling Dictionaries</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="availDictTableWidget">
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="dlLabel">
           <property name="text">
            <string>Downloading</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="downloadProgressBar">
           <property name="value">
            <number>0</number>
           </property>
           <property name="textVisible">
            <bool>true</bool>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="availListDownloadButton">
           <property name="text">
            <string>Refresh Downloadable List</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="spellDownloadButton">
           <property name="text">
            <string>Download Selected Dictionaries</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="5" margin="10"/>
 <resources/>
 <connections/>
</ui>
